language/move-prover/doc/user/spec-lang.md

```diff
@@ -631,7 +644,7 @@ whether or not `mutate_at` is inlined.
 
 If the modifies annotation is omitted on a function, then that function is deemed to have all
 possible permissions for those resources it may modify during its execution.  The set of all
-resources that may be modified by a function is obtained via an interprocedural analysis of
+resources that may be modified by a function is obtained via an inter-procedural analysis of
```

Comment:
This is not usually hyphenated, in my experience

Reply:
Done